docs/documentation/concepts/items.md

```diff
@@ -28,11 +28,43 @@ The following item types are currently available (alphabetical order):
 | String         | Stores texts | String |
 | Switch         | Typically used for lights (on/off) | OnOff |
 
-Group Items can derive their own state depending on their member items.
-
-- AVG displays the average of the item states in the group.
-- OR displays an OR of the group, typically used to display whether any item in a group has been set.
-- other aggregations: AND, SUM, MIN, MAX, NAND, NOR
+## Group Items
+
+Group items collect other items into groups.
+Group items can themselves be members of other group items.
+Cyclic membership is not forbidden but strongly not recommended.
+User interfaces might display group items as single entries and provide navigation to its members.
+
+Example for a Group item simply collection other items:
+```
+    Group groundFloor
+    Switch kitchenLight (groundFloor)
+    Switch livingroomLight (groundFloor)
+``` 
+
+In addition Group items can calculate their own state derived from their members using a specific group function.
```

Comment:
-> "In addition, "

Reply:
I would actually remove the whole sentence - this content comes in the next section again.